scalableminds/webknossos
# Disable settings that need a page reload (when logged out)

...since the changed setting wouldn't be stored, otherwise.

### URL of deployed dev instance (used for testing):
- https://disablereloadsettings.webknossos.xyz

### Steps to test:
- make a DS public
- log out and visit that DS
- try to change the GPU rendering mode or the antialiasing setting -> should be disabled with a tooltip
- log in and try to change the settings -> should work and no tooltip should be visible

### Issues:
- fixes https://scm.slack.com/archives/C5AKLAV0B/p1689867565444779
